script/GameObjects/Components/Economy/Economy.lua

```diff
@@ -38,15 +39,28 @@ function Economy:update (dt)
       end
     end
 
+    -- Cache profitable mining jobs
+    local alljobcount = 0
+    local realjobcount = 0
```

Comment:
realJobCount

Reply:
Change made, update in follow-up commit.